bugsnag-android-core/src/main/java/com/bugsnag/android/Stackframe.kt

```diff
@@ -128,9 +128,9 @@ class Stackframe : JsonStream.Streamable {
 
         writer.name("columnNumber").value(columnNumber)
 
-        frameAddress?.let { writer.name("frameAddress").value(it) }
-        symbolAddress?.let { writer.name("symbolAddress").value(it) }
-        loadAddress?.let { writer.name("loadAddress").value(it) }
+        frameAddress?.let { writer.name("frameAddress").value("0x%x".format(it)) }
+        symbolAddress?.let { writer.name("symbolAddress").value("0x%x".format(it)) }
+        loadAddress?.let { writer.name("loadAddress").value("0x%x".format(it)) }
```

Comment:
This should probably use a specialised formatter to ensure that the outputted values are always output in an unsigned format.